gettsim/transfers/wohngeld.py

```diff
@@ -489,7 +489,7 @@ def wohngeld_miete_ab_2009(
     return wg_miete
 
 
-def wohngeld_max_miete_ab_2021(
+def wohngeld_miete_ab_2021(
```

Comment:
Docstring still says 2009. 

Why are there no tests for this set of functions? Seems reasonable easy that an error would slip in...

Given that the PR starts with that calculator, taking a couple of those examples including extreme ones / corner cases (hh_size in {5, 6, 13}) and making test cases out of them would seem like the natural place to start.

Reply:
Docstring is corrected.

There are two tests for 2021. The one I describe above and `test_increasing_hh_size()` is extended for 2021. To me, this confirms that the logic of the 2021 reform is captured correctly as it does differs only in the maximum value for the considered rent. Extreme cases (hh_size > 12) are an issue that would affect all wohngeld calculations and would deserve an own PR in my view. @mjbloemer has played around with wohngeld calculations and very large households.

Reply:
Okay, I was just surprised that the renaming of the function did not lead
to renaming anywhere in the tests.

In the long run, it would be good to also test functions like this
directly, but no prerequisite for merging this IMO.

Reply:
Testing each function directly will also make it easier to catch side-effects, see #272